tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -141,17 +143,28 @@ class ConfigurationPassImpl @Inject() (
       module: TlaModule,
       outOptions: WriteablePassOptions): Seq[TlaDecl] = {
     var configuredModule = module
-    // read TLC config if present
+    // read a TLC config if it was passed by the user
     val configFilename = options.getOrElse[String]("checker", "config", "")
-    val filename =
-      if (configFilename.isEmpty) {
-        options
-          .getOrError[String]("parser", "filename")
-          .replaceFirst("\\.(tla|json)$", "\\.cfg")
-      } else {
-        configFilename
+    if (configFilename.isEmpty) {
+      // The older versions of apalache were loading a TLC config file of the same basename as the spec.
+      // We have flipped this behavior in version 0.25.0.
+      // Hence, warn the user that their config is not loaded by default.
+      val stem = FilenameUtils.removeExtension(options.getOrError[String]("parser", "filename"))
+      val defaultConfig = new File(stem + ".cfg")
+      if (defaultConfig.exists()) {
+        val msg = s"  > Ignoring the accompanying TLC config file. To enable it, pass --config=$defaultConfig."
+        logger.info(msg)
       }
-    val basename = FilenameUtils.getName(filename)
+      // return immediately without going all the way down
+      return List()
```

Comment:
I had to jump up to the Scaladoc to understand what `List()` is here, so probably this is better:

```suggestion
      // return immediately, with no additional declarations
      return List()
```